packages/jest-circus/src/utils.js

```diff
@@ -243,16 +248,25 @@ const makeTestResults = (describeBlock: DescribeBlock): TestResults => {
     if (!status) {
       throw new Error('Status should be present after tests are run.');
     }
+
+    let location = null;
```

Comment:
I wonder if this needs to be undefined?

Reply:
The test expliclity asserts `.toBeNull()`.

https://github.com/facebook/jest/blob/master/integration-tests/__tests__/location_in_results.test.js#L22-L23